ironfish/src/assets/assetsVerifier.ts

```diff
@@ -0,0 +1,81 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { createRootLogger, Logger } from '../logger'
+import { ErrorUtils } from '../utils'
+import { SetIntervalToken } from '../utils'
+import { AssetsVerificationApi, VerifiedAssets } from './assetsVerificationApi'
+
+export type AssetVerification = {
+  status: 'verified' | 'unverified' | 'unknown'
+}
+
```

Comment:
nit: The nomenclature is a bit misleading here, this doesn't actually do any asset validation, it just retrieves verified assets, maybe rename to something like `VerifiedAssets`

Reply:
This comment is on the `AssetVerification` type, but I think you're referring to the `AssetsVerifier` class and the module name, am I right?

Reply:
Yea I am, weird the comment looks like it is on line 12/13 to me.

Reply:
I think you have a valid point here. I'll try to think about better names, but I think what you propose (`VerifiedAssets`) is consistent with `Telemetry`, so I might just actually go with your suggestion.

Reply:
And also the `verify` method should change name. Maybe `classify` is a good candidate?

Reply:
I think both the current name, or VerifiedAssets is fine to me. Though I personally like `verify()`. What does the verifier do? verify.

Reply:
I will keep the current names as there is no clear consensus. I might come back to it in a future PR